readme.md

```diff
@@ -62,14 +63,13 @@ If you want to see your changes, just go to your label settings on GitHub to see
 ### Duplicate labels of an existing repository
 
 1. Install label-manager on the repository with the labels you want to duplicate
```

Comment:
Hmm I would remove the list items and just mention that you can simply copy-paste the config to another repository, remove the `id`s and synchronise it.